experimental/python/pyproject.toml

```diff
@@ -9,7 +9,7 @@ authors = [
 
 readme = "README.md"
 requires-python = ">=3.10"
-license = {file = "LICENSE"}
+license = { name = "Databricks License", file = "LICENSE" }
```

Comment:
Could you refer to the root one directly and remove the local file?

Reply:
We will have to create a symlink, because you can't link files from parent directory

Reply:
I changed to Symlink, but I'm not sure how it works on Windows. I've manually checked that wheel is packaged correctly.